graphics/src/ColladaExporter.cc

```diff
@@ -152,6 +153,14 @@ ColladaExporter::~ColladaExporter()
 //////////////////////////////////////////////////
 void ColladaExporter::Export(const Mesh *_mesh, const std::string &_filename,
     bool _exportTextures)
+{
+  std::vector<math::Matrix4d> empty;
+  this->ColladaExporter::Export(_mesh, _filename, _exportTextures, empty);
+}
+
+//////////////////////////////////////////////////
+void ColladaExporter::Export(const Mesh *_mesh, const std::string &_filename,
+    bool _exportTextures, std::vector<math::Matrix4d> &_submeshToMatrix)
```

Comment:
does the `_submeshToMatrix` size need to match the `_mesh->SubMeshCount()`? If so, we should add a check and print out an error if it does not